.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+    "beautify.tabSize": 2
```

Comment:
This is a setting for visualstudio code. I could probably add all of .vscode to .gitignore.